Update README.md

styx-techno opened the pull request:
Update for 64-bit support and build. Display is now semi-broken and the canvas gets shown in the right side bar with a wrong resolution, not in the full screen display.

styx-techno committed:
Update README.md

Update for 64-bit support and build. Display is now semi-broken and the canvas gets shown in the right side bar with a wrong resolution, not in the full screen display.

itsmeow commented:
The full screen and npv depend on your config. If it's not showing in fullscreen at all the CSS query probably just needs an update. If Spotify runs on 64bit CEF now we need to update the general instructions and not just the quick script

styx-techno committed:
Update README.md

had the time to go through the readme in full, was a bit late last night when I created the pull request. You were absolutely right.

itsmeow commented:
Instead of adding "EDIT:" to stuff, just update it entirely. Also remove the spaces in `is_component_build = true`

styx-techno committed:
Update README.md

itsmeow commented on README.md:
```diff
@@ -128,18 +121,17 @@ Next, download [automate-git.py](https://bitbucket.org/chromiumembedded/cef/raw/
 set CEF_USE_GN=1
-set GN_DEFINES=is_official_build=true proprietary_codecs=true ffmpeg_branding=Chrome use_thin_lto=false enable_nacl=false blink_symbol_level=0 symbol_level=0
-set GYP_MSVS_VERSION=2019
+set GN_DEFINES=is_official_build=true is_component_build=true proprietary_codecs=true ffmpeg_branding=Chrome use_thin_lto=false target_cpu ="x64" enable_nacl=false blink_symbol_level=0 symbol_level=0
```
```suggestion
set GN_DEFINES=is_official_build=true is_component_build=true proprietary_codecs=true ffmpeg_branding=Chrome use_thin_lto=false target_cpu="x64" enable_nacl=false blink_symbol_level=0 symbol_level=0
```

itsmeow commented on README.md:
```diff
@@ -204,7 +196,7 @@ Next, you will need to back up your old CEF assets. Create a folder called `back
 - chrome_100_percent.pak
 - chrome_200_percent.pak
-- cef_extensions.pak
+- cef_extensions.pak EDIT: Does not exist anymore
```
```suggestion
```

itsmeow commented on README.md:
```diff
@@ -252,24 +241,25 @@ set "output=%output:~1%"
 
 set CEF_USE_GN=1
-set GN_DEFINES=is_official_build=true proprietary_codecs=true ffmpeg_branding=Chrome use_thin_lto=false enable_nacl=false blink_symbol_level=0 symbol_level=0
+set GN_DEFINES=is_official_build=true is_component_build=true proprietary_codecs=true ffmpeg_branding=Chrome use_thin_lto=false target_cpu ="x64" enable_nacl=false blink_symbol_level=0 symbol_level=0
```
```suggestion
set GN_DEFINES=is_official_build=true is_component_build=true proprietary_codecs=true ffmpeg_branding=Chrome use_thin_lto=false target_cpu="x64" enable_nacl=false blink_symbol_level=0 symbol_level=0
```

itsmeow committed:
Update README.md

itsmeow committed:
Update README.md

itsmeow committed:
Update README.md

itsmeow commented on README.md:
```diff
@@ -128,18 +121,17 @@ Next, download [automate-git.py](https://bitbucket.org/chromiumembedded/cef/raw/
 
 Create a batch script inside `C:\code\`, name it `build.bat`. If the name change doesn't change the icon, press View in file explorer and check "File name extensions" and rename it again. Paste the following into this script and save it (you can open it by right clicking and pressing edit):
 
+Spotify recently switched to 64-bit, you can check this in your spotify app by clicking on the three menu dots in the top left, help, about Spotify. The following intsructions will continue wtih the assumption that you want to build cef for 64-bit.
```
```suggestion
Spotify recently switched to 64-bit, you can check this in your Spotify app by clicking on the three menu dots in the top left, then Help -> About Spotify. The following instructions will continue with the assumption that you want to build CEF for 64-bit.
```

itsmeow committed:
Update README.md

itsmeow commented on README.md:
````diff
@@ -213,25 +204,22 @@ Next, you will need to back up your old CEF assets. Create a folder called `back
+If you made it this far, you are truly technically competent! Congratulations.
 
 Now that you have shown that you are truly competent, here is a little lazy upgrade script that you can use inside your build.bat:
 ```
````
```suggestion
```batch
```

itsmeow committed:
Update README.md
